recipes/jsonschema2md/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "jsonschema2md" %}
+{% set version = "v0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/RalfG/jsonschema2md/archive/refs/tags/v0.4.0.tar.gz
+  sha256: 12a7ffb946cd5f8f452dbde0fca83dba9b03a039bce6596f030b416e5162fc52
+
+build:
```

Comment:
This seems like it could be noarch: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python